changelogs/current_change_log.md

```diff
@@ -4,9 +4,27 @@
 
 ---
 
+### Tables changed
+
+Table           | Attributes added | Attributes removed | Renamed
+:-- |:-- |:--
+PatientCondition | addedDatetime | addedDateTime | ✓
+PatientCondition | resolutionDateTime |resolutionDatetime | ✓
+ConsultationRequest | statusChangeTime |statusChangeDatetime | ✓
+HospitalVisit | presentationDatetime | presentationTime | ✓
+HospitalVisit | admissionDatetime | admissionTime | ✓
+HospitalVisit | dischargeDatetime | dischargeTime | ✓
+LabSample | receiptAtLabDatetime | receiptAtLab | ✓
+LabSample | sampleCollectionDatetime | sampleCollectionTime | ✓
+LocationVisit | admissionDatetime | admissionTime | ✓
+LocationVisit | dischargeDatetime | dischargeTime | ✓
+VisitObservationType | creationDatetime | creationTime | ✓
```

Comment:
worth having these as their SQL representation so:
`patient_condition | added_datetime | added_date_tiime`

Though I don't imagine we've been consistent with this - we are releasing a database so should use what the end user will be seeing

Reply:
agreed!